recipe/meta.yaml

```
@@ -32,14 +32,13 @@ requirements:
     - packaging >=23.0
     - pathspec >=0.9.0
     - plumbum >=1.6.9
-    # TODO: remove when fixed upstream https://github.com/copier-org/copier/issues/1225
-    - pydantic >=1.10.2,<2
+    - pydantic >=2.0.3
     - pygments >=2.7.1
-    - python >=3.8
+    - python >=3.8,<4.0
     - pyyaml >=5.3.1
     - pyyaml-include >=1.2
     - questionary >=1.8.1
-    - git >=2.27
+    - typing-extensions >=3.7.4,<5.0.0
```

Same here - git is available on conda (not on PyPI). Please don't change pure conda dependencies in upgrade PR except if needed - these changes must be discussed as they may break consumer workflow.

```suggestion
    - git >=2.27
    - typing-extensions >=3.7.4,<5.0.0
```